GridTariffAPI/StartupTasks/SyncMeteringPointTariffStartupTask.cs

```diff
@@ -0,0 +1,32 @@
+﻿using GridTariffApi.BigQuery.MeteringPointTariffSync;
+using GridTariffApi.Database;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace GridTariffApi.StartupTasks
+{
+    public class SyncMeteringPointTariffStartupTask : IStartupTask
+    {
+        private readonly IServiceProvider _serviceProvider;
+        public SyncMeteringPointTariffStartupTask(IServiceProvider serviceProvider)
+        {
+            _serviceProvider = serviceProvider;
+
+        }
+        public int GetExecutionOrder()
+        {
+            return 2;
+        }
+
+        public async Task Execute()
+        {
+            using var scope = _serviceProvider.CreateScope();
+            var elviaDbContext = scope.ServiceProvider.GetRequiredService<ElviaDbContext>();
+            var meteringPointTariffSynchronizer = scope.ServiceProvider.GetRequiredService<IMeteringPointTariffSynchronizer>();
+            var elviaCompany = elviaDbContext.Company.FirstOrDefault();
+            await meteringPointTariffSynchronizer.SynchronizeMeteringPointsAsync(elviaDbContext, elviaCompany, DateTimeOffset.UtcNow);
```

Comment:
Fint å få gjort dette ved aller første gangs oppstart. Men kanskje ikke nødvendig å gjøre det hver gang en pod restartes? Kanskje legge inn en sjekk på om en tabeller er tom?

Reply:
meteringPointTariffSynchronizer.SynchronizeMeteringPointsAsync har fått plassert ansvar for å avklare om det skal utføres fullsync eller inkrementell sync. 
Døgnlig inkrementell sync bruker ikke mange sekunder, og en inkrementell sync basert på restart av pod tar kun kort tid.
Lar denne være.